src/aeso_ast_infer_types.erl

```diff
@@ -229,7 +229,7 @@ force_bind_fun(X, Type, Env = #env{ what = What }) ->
     NoCode = get_option(no_code, false),
     Entry = if X == "init", What == contract, not NoCode ->
                     {reserved_init, Ann, Type};
-               What == contract_interface -> {contract_fun, Ann, Type};
+               What == contract orelse What == contract_interface -> {contract_fun, Ann, Type};
```

Comment:
Mixing style, the previous clause uses `,` here we use `orelse` - consider `;`?

Reply:
Done here 3838a47694c2db10663ff79b7f54d4667e3fb9e1